shared_model/backend/protobuf/queries/impl/proto_get_account_assets.cpp

```diff
@@ -30,5 +39,12 @@ namespace shared_model {
       return account_assets_.account_id();
     }
 
+    boost::optional<const interface::AccountAssetPaginationMeta &>
+    GetAccountAssets::paginationMeta() const {
+      return pagination_meta_
```

Comment:
Not sure if using a ternary op is better than a plain conditional here, because a plain conditional will allow to use `boost::none`.